Src/DependencyCollector/Shared/Implementation/DependencyTableStore.cs

```diff
@@ -35,6 +35,7 @@ internal static DependencyTableStore Instance
 
         public void Dispose()
         {
+            instance = null;
```

Comment:
Its singleton for a reason and I am not changing it. Dipose() will simply call the dispose of the Disposable instances it created.